lib/build.js

```diff
@@ -83,39 +83,62 @@ class BuildModel extends BaseModel {
             number: now
         });
         const initialBuildData = {
-            cause: 'Started by user',
+            cause: `Started by user ${config.username}`,
```

Comment:
It is cool, but it's not correct sadly.  Username is for looking up which GitHub token to use.  Maybe we should rename username and pass in it as another value.